Brocfile.js

```diff
@@ -0,0 +1,35 @@
+/* global require, module */
+
+var EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
+var fs = require('fs');
+
+// TODO(azirbel): Use something more legit
+function htmlEntities(str) {
+  return String(str).replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/>/g, '&gt;').replace(/"/g, '&quot;');
+}
+
+var app = new EmberAddon({
+  /*
+   * Replace patterns. We use this to replace strings such as:
+   * @@{controllers/file.js}
+   * With the content of those files.
+   */
+  replace: {
+    files: [
+      '**/*.js'
+    ],
+    patterns: [{
+      match: /@@{[^}]*}/g,
+      replacement: function(matchedText) {
+        filename = matchedText.slice(3, -1);
```

Comment:
@cyril-sf we need some easy way for folks to sanity check their changes. 

Maybe eventually that means having a separate demo app repo that you can `link` to this addon or something, but for now I think we should leave it here so you can just `ember server`.